packages/eslint-plugin/src/rules/unbound-method.ts

```diff
@@ -18,6 +18,59 @@ export type Options = [Config];
 
 export type MessageIds = 'unbound';
 
+const nativelyBoundMembers = ([
+  'Promise',
+  'Number',
+  'Object',
+  'String', // eslint-disable-line @typescript-eslint/internal/prefer-ast-types-enum
+  'RegExp',
+  'Symbol',
+  'Array',
+  'Proxy',
+  'Date',
+  'Infinity',
```

Comment:
`Infinity` has no own methods at all, it's not an object? And all the methods inherited from `Number.prototype` are not bound.

Reply:
`Number.prototype` methods aren't included.
Only static members are included.

I.e. `Number.isFinite()`, `Number.isNaN()`

Reply:
Sure, static `Number` methods are fine. But the inclusion of `Infinity` in the list means that `Infinity.toString`, `Infinity.toFixed` etc would be included, right? They certainly shouldn't be.